src/shorthands/JBox.jl

```diff
@@ -0,0 +1,53 @@
+function _JBox(cornerpoint1::Point, cornerpoint2::Point, color::String, action::Symbol, vertices::Bool)
+    sethue(color)
+    box(cornerpoint1, cornerpoint2, action, vertices=vertices)
+end
+function _JBox(points::Array, color::String, action::Symbol, vertices::Bool)
+    sethue(color)
+    box(points, action, vertices=vertices)
+end
+function _JBox(pt::Point, width::Real, height::Real, color::String, action::Symbol, vertices::Bool)
+    sethue(color)
+    box(pt, width, height, action, vertices=vertices)
+end
+function _JBox(pt::Point, width::Real, height::Real, cornerradius::Float64, color::String, action::Symbol)
+    sethue(color)
+    box(pt, width, height, cornerradius, action=action)
+end
+
+"""
+    JBox(cornerpoint1::Point, cornerpoint2::Point; color="black", action=:stroke, vertices=false)
+
+Create a box (rectangle) between two points and do an action.
+Use vertices=true to return an array of the four corner points: bottom left, top left, top right, bottom right.
+"""
+JBox(cornerpoint1::Point, cornerpoint2::Point; color="black", action=:stroke, vertices=false) = (args...) -> _JBox(cornerpoint1, cornerpoint2, color, action, vertices)
```

Comment:
This currently limits the functionality as one can't use the `change` Action, as that needs keyword arguments, right?
I think:
```julia
JBox(cornerpoint1::Point, cornerpoint2::Point; color="black", action=:stroke, vertices=false) = (args...; kcolor=color, kaction=action, kvertices=vertices) -> _JBox(cornerpoint1, cornerpoint2, kcolor, kaction, kvertices)
```
should work? 
Maybe even have `cornerpoint` in there to have full flexibility.

Reply:
So this needs to be done for all the shapes right?

Reply:
Yes :wink:

Reply:
Okay So this works!
Is there any specific reason for having "k" pre-fixed to the argument names.
` kcolor=color, kaction=action, kvertices=vertices) `

I tried with `color=color, action=action, vertices=vertices) ` and it works.

Reply:
I was wondering which one is taken then but as it's a different scope it's probably fine :smile: